src/commands/app/create.ts

```diff
@@ -120,6 +132,12 @@ export default class Create extends BaseCommand<typeof Create> {
     await this.unZipBoilerplate(await this.cloneBoilerplate());
     tmp.setGracefulCleanup(); // NOTE If graceful cleanup is set, tmp will remove all controlled temporary objects on process exit
 
+    // To remove the default app name from flag and replace it with the actual folder name
+    this.sharedConfig.appName =
+      this.sharedConfig.folderPath.split("/").pop() ||
+      this.sharedConfig.appName;
+    this.tempAppData.name = this.sharedConfig.appName;
```

Comment:
@harshithad0703, pls verify and remove this code if not required.

Reply:
removed these lines